sample_workloads/lit-gpt-demo/helm/values.yaml

```diff
@@ -1,22 +1,25 @@
 cluster:
-  nNodes: 8
+  nNodes: 4
```

Comment:
want to double check here, do we expect this value to be 4?

Reply:
Yep, defaulting to 4.